packages/kujira-fin/src/query.rs

```diff
@@ -86,6 +95,43 @@ pub struct OrderResponse {
     pub original_offer_amount: Uint256,
 }
 
+#[cw_serde]
+pub struct OrderResponseOracle {
+    /// A unique ID for the order
+    pub idx: Uint128,
+
+    /// The address used to place the order
+    pub owner: Addr,
+
+    /// The delta in basis points from the current oracle price
+    pub delta: i8,
+
+    /// The price of the order based on current oracle values
+    pub price: Decimal256,
+
+    /// The denom offered
+    pub offer_denom: Denom,
+
+    /// The remaining order amount
+    pub offer_amount: Uint256,
+
+    /// Amount of filled order awaiting withdrawal
+    pub filled_amount: Uint256,
+
+    /// Timestamp of original creation
+    pub created_at: Timestamp,
+
+    /// Offer amount at time of creation
+    pub original_offer_amount: Uint256,
+}
+
+#[cw_serde]
+#[serde(untagged)]
+pub enum OrderResponse {
```

Comment:
Does this have the potential to break contracts that are querying arbitrary orders using their `idx`, because of the addition of all the different fields (as is necessary)

Can't think of any live contract that would break on this behavior off the top of my head though.